config_defaults.yml

```diff
@@ -1,29 +1,72 @@
 ---
 .travis.yml:
-  script: "\"bundle exec rake validate && bundle exec rake lint && bundle exec rake spec SPEC_OPTS='--format documentation'\""
+  supported_oss: &supported_oss
+    puppetlabs-acl:                      [windows]
+    puppetlabs-chocolatey:               [windows]
+    puppetlabs-dism:                     [windows]
+    puppetlabs-dsc:                      [windows]
+    puppetlabs-iis:                      [windows]
+    puppetlabs-powershell:               [windows]
+    puppetlabs-reboot:                   [windows]
+    puppetlabs-registry:                 [windows]
+    puppetlabs-sqlserver:                [windows]
+    puppetlabs-wsus_client:              [windows]
+    puppetlabs-inifile:                  [linux,windows]
+    puppetlabs-stdlib:                   [linux,windows]
+    puppetlabs-accounts:                 [linux]
+    puppetlabs-apache:                   [linux]
+    puppetlabs-apt:                      [linux]
+    puppetlabs-concat:                   [linux]
+    puppetlabs-firewall:                 [linux]
+    puppetlabs-haproxy:                  [linux]
+    puppetlabs-hocon:                    [linux]
+    puppetlabs-ibm_installation_manager: [linux]
+    puppetlabs-inifile:                  [linux]
+    puppetlabs-java:                     [linux]
+    puppetlabs-java_ks:                  [linux]
+    puppetlabs-mongodb:                  [linux]
+    puppetlabs-motd:                     [linux]
+    puppetlabs-mysql:                    [linux]
+    puppetlabs-ntp:                      [linux]
+    puppetlabs-postgresql:               [linux]
+    puppetlabs-rabbitmq:                 [linux]
+    puppetlabs-stdlib:                   [linux]
+    puppetlabs-tagmail:                  [linux]
+    puppetlabs-tftp:                     [linux]
+    puppetlabs-tomcat:                   [linux]
+    puppetlabs-vcsrepo:                  [linux]
+    puppetlabs-vsphere:                  [linux]
+    puppetlabs-xinetd:                   [linux]
```

Comment:
I originally had it as a top-level hash key called `supported_oss` that I could anchor around the yaml file, but msync assumes that all top-level keys are files and can't find a file called `moduleroot/supported_oss` and fails, so I moved it into travis arbitrarily and reference it elsewhere...

It's not great (and honestly I don't like supported_oss anyway because managed_modules.yml duplicates this basically, but that can't hold other data) but can't find a better way yet.